### PR TITLE
diesel engine screenshot

### PR DESCRIPTION
**Pull request by evanrivera**
i will be working on the diesel engine portion of the class project. thanks!

### CODE REVIEW

File: DieselEngineAssembly [hunk 1]
```diff
@@ -0,0 +1,5 @@
+private class DieselEngineAssembly
```
Comment (anthonykulis) [inline DieselEngineAssembly]: do not write UML as code and commit it. Thats the point of UML. No need to repeat it.

**Comment by KenanJones** [comment]
You can look at my work to see the general pattern, but since you only have one instantiable class, you will have to combine the pieces from all my classes and change the "sport" to "diesel" where necessary.

**Comment by anthonykulis** [comment]
uml?

File: edu/jalc/automobile/onlinebuilder/components/engine/diesel/DieselEngineAssemblyTest.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+package edu.jalc.automobile.onlinebuilder.components.engine.diesel
+
+import edu.jalc.automobile.onlinebuilder.components.engine.specs.*;
+
+public class DieselEngineAssemblyTest{
```
Comment (KenanJones) [inline edu/jalc/automobile/onlinebuilder/components/engine/diesel/DieselEngineAssemblyTest.java]: This should be DieselEngineAssemblyTest, but the code is testing DieselEngine. I should have explained the difference better to you when you joined our team. I'm sorry. You need to test engine, exhaust,and induction.

**Comment by anthonykulis** [comment]
No point now. It's past due. You need to merge his branch in so I can merge
yours

**Pull request merged**